File: src/main/java/com/example/installcertatruntime/car/CarController.java
package com.example.installcertatruntime.car;

import org.springframework.web.bind.annotation.*;

import java.util.List;

@RestController
@RequestMapping(value = "api/v1/cars")
public class CarController {

    private final CarService carService;

    public CarController(CarService carService) {
        this.carService = carService;
    }

    @GetMapping("")
    public List<Car> all() {
        return carService.all();

    }

    @PostMapping("")
    public Car create(@RequestBody Car car) {
        return carService.create(car);
    }

}

File: src/main/kotlin/com/example/installcertatruntime/car/CarController.kt
package com.example.installcertatruntime.car

import org.springframework.web.bind.annotation.*

@RestController
@RequestMapping("api/v1/cars")
class CarController(private val carService: CarService) {

    @GetMapping("")
    fun all(): List<*> {
        return carService.all()
    }

    @PostMapping("")
    fun create(@RequestBody car: Car): Car {
        return carService.create(car)
    }

}
